### PR TITLE
Remove Feedback functionality. Updated README. Setup env prop.

### PR DESCRIPTION
**Pull request by vglushak-vt**

### CODE REVIEW

File: api/app/Module.scala [hunk 1]
```diff
@@ -29,11 +29,16 @@ class Module(environment: Environment,
...
 
   override def configure() = {
-    //val esUri = configuration.getString("elasticsearch.uri").getOrElse("elasticsearch://localhost:9300")
-    //Testing change 06/12/2016
-    val esUri = configuration.getString("elasticsearch.uri").getOrElse("elasticsearch://10.50.33.20:9300")
+    val env = sys.props.get("environment").getOrElse(sys.error("Setup system property 'environment'"))
 
-    val esCluster = configuration.getString("elasticsearch.cluster.name").getOrElse("elasticsearch_" + System.getProperty("user.name"))
+    val envConf = configuration.getConfig(s"env.$env") match {
+      case None => sys.error(s"Unable to find config for '$env' env")
```
Comment (iRhysBradbury) [inline api/app/Module.scala]: envConf type not defined, could be either a `Unit` or `Config` at runtime. 

You want to provide a default config, utilise the monad.


eg:

```
configuration.getConfig("x").getOrElse {
sys.error("some error message")
Config(//default config)
}
```

File: api/app/Module.scala [hunk 2]
```diff
@@ -29,11 +29,16 @@ class Module(environment: Environment,
...
+      case Some(c) => c
+    }
+
+    val esUri = envConf.getString("elasticsearch.uri").get
```
Comment (iRhysBradbury) [inline api/app/Module.scala]: Dont use `.get` on options. Always provide default.
eg:

```
envConf.getString("elasticsearch.uri").getOrElse("SomeDefaultURI")
```

File: api/app/Module.scala [hunk 3]
```diff
@@ -29,11 +29,16 @@ class Module(environment: Environment,
...
+
+    val esCluster = envConf.getString("elasticsearch.cluster.name").getOrElse("elasticsearch_" + System.getProperty("user.name"))
```
Comment (iRhysBradbury) [inline api/app/Module.scala]: Why is user name is here?

File: api/app/Module.scala (hunk 1 above)
Comment (iRhysBradbury) [inline api/app/Module.scala]: We use pure config in AddressIndex REF: https://github.com/ONSdigital/address-index-api/blob/develop/server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexConfigModule.scala#L16

Comment (vglushak-vt) [inline api/app/Module.scala]: How can it be Unit?

File: api/app/Module.scala (hunk 3 above)
Comment (vglushak-vt) [inline api/app/Module.scala]: Don't know. I've just change the name of variable used.

File: api/app/Module.scala (hunk 2 above)
Comment (vglushak-vt) [inline api/app/Module.scala]: Why do we want default url, which might make no sense for other env (for example there was some dev url before which makes not much sense for local env neither for prod). 
I would prefer error on startup time - to see the problem earlier and fix configs properly.

File: api/app/Module.scala (hunk 1 above)
Comment (iRhysBradbury) [inline api/app/Module.scala]: `case None`

File: api/app/Module.scala (hunk 2 above)
Comment (iRhysBradbury) [inline api/app/Module.scala]: If your requirement is to halt the app when config isn't found, then ignore this feedback.

File: api/app/Module.scala (hunk 1 above)
Comment (vglushak-vt) [inline api/app/Module.scala]: `sys.error` is throwing exception, it's never return value.
Type inference in IDEA shows that return type is `Configuration`.
So it should not be a problem.

File: api/app/Module.scala (hunk 2 above)
Comment (iRhysBradbury) [inline api/app/Module.scala]: This `.get` line could lead to runtime errors

Comment (iRhysBradbury) [inline api/app/Module.scala]: Scenario:
app.conf:
```
env.$env {
  someDummyConfLine =5
}

```

This `.get` will RuntimeException.

Comment (vglushak-vt) [inline api/app/Module.scala]: Let me fix it.

File: api/app/Module.scala (hunk 3 above)
Comment (onscmhw) [inline api/app/Module.scala]: If you run ElasticSearch locally e.g. for development, it creates the default ES server with a name like "elasticsearch_myname".

File: api/app/Module.scala (hunk 1 above)
Comment (iRhysBradbury) [inline api/app/Module.scala]: suggestion:
same outcome less code:
`configuration.getConfig(s"env.$env").getOrElse(sys.error(s"Unable to find config for '$env' env"))`

File: api/app/Module.scala (hunk 3 above)
Comment (iRhysBradbury) [inline api/app/Module.scala]: are you guys using vagrant setup?

**Pull request merged**